packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -1732,6 +1732,8 @@ export interface _IdentifierProviderProps {
 
 // @internal
 export interface _Identifiers {
+    atMentionSuggestionItem: string;
```

Comment:
I guess these should not be part of stable change?

Reply:
@PorterNan  These are needed in _AtMentionFlyout and since we're not removing _AtMentionFlyout using a feature flag, we cannot remove these. I tried to put a beta comment for these but seems like `rush build` still add them on stable. Any other way I can remove these from stable? Thanks